app/lib/services/funding_eligibility.rb

```diff
@@ -88,9 +88,24 @@ def funding_eligiblity_status_code
       end
     end
 
-    def eligible_for_targeted_delivery_funding?
-      course_and_institution_eligible_for_targeted_delivery_funding? &&
-        !previously_received_targeted_funding_support? # Check last as it involves an API call
+    def tsf_eligibility_calculator
+      # This returns the tsf calculation plus the previously received funding result from ECF:
```

Comment:
I don't think this is accurate since previously_received_targeted_funding_support? is taken into account outside this class now.

Reply:
yeah need to sort the comment i guess is what you mean?